.github/workflows/released.yml

```diff
@@ -0,0 +1,33 @@
+name: Merge trunk to develop branch and update tag
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  update:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.release.target_commitish }}
+
+      - name: Merge trunk to develop branch
+        uses: devmasx/merge-branch@v1.4.0
+        with:
+          type: now
+          from_branch: trunk
+          target_branch: develop
+          github_token: ${{ github.token }}
+
+      - name: Force push ${{ github.event.release.tag_name }} tag to latest commit in trunk
+        run: |
+          SHA=$(git rev-parse trunk)
+          TAG=$(git describe --tags --abbrev=0)
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Could we have this email as a parameter, maybe an environment varialbe? Also, I have the feeling that this hardcoded value will not work

Reply:
About the email value, look [here](https://github.community/t/github-actions-bot-email-address/17204/6) and [here](https://api.github.com/users/github-actions%5Bbot%5D). I can also confirm that it works for my test repo.

Having said that, we could opt for removing the email. However I don't know if this will be an option if we want to sign bot commits.

I didn't think of putting this as an env variable, as it is only used in one place. Let me know if you want me to do this, personally I don't see a need for it currently.

Reply:
I was suspecting that the email would change in different users/repos.

It is generally a bad idea to have hardcoded values like that in code. It lacks flexibility (imagine we need to change it) and we lack the ability to obfuscate values.